
From: Dipankar Sarma <dipankar@in.ibm.com>

Somehow spaces replaced tabs in rcupdate.h and I would like to keep
everything clean.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/rcupdate.h |   18 +++++++++---------
 1 files changed, 9 insertions(+), 9 deletions(-)

diff -puN include/linux/rcupdate.h~rcu-fix-spaces-in-rcupdateh include/linux/rcupdate.h
--- 25/include/linux/rcupdate.h~rcu-fix-spaces-in-rcupdateh	2004-08-07 02:07:34.259070664 -0700
+++ 25-akpm/include/linux/rcupdate.h	2004-08-07 02:07:34.262070208 -0700
@@ -87,20 +87,20 @@ static inline int rcu_batch_after(long a
  */
 struct rcu_data {
 	/* 1) quiescent state handling : */
-        long		quiescbatch;     /* Batch # for grace period */
+	long		quiescbatch;     /* Batch # for grace period */
 	long		qsctr;		 /* User-mode/idle loop etc. */
-        long            last_qsctr;	 /* value of qsctr at beginning */
-                                         /* of rcu grace period */
+	long            last_qsctr;	 /* value of qsctr at beginning */
+					 /* of rcu grace period */
 	int		qs_pending;	 /* core waits for quiesc state */
 
 	/* 2) batch handling */
-        long  	       	batch;           /* Batch # for current RCU batch */
-        struct rcu_head *nxtlist;
+	long  	       	batch;           /* Batch # for current RCU batch */
+	struct rcu_head *nxtlist;
 	struct rcu_head **nxttail;
-        struct rcu_head *curlist;
-        struct rcu_head **curtail;
-        struct rcu_head *donelist;
-        struct rcu_head **donetail;
+	struct rcu_head *curlist;
+	struct rcu_head **curtail;
+	struct rcu_head *donelist;
+	struct rcu_head **donetail;
 	int cpu;
 };
 
_
